[2023-06-15T11:25:01.449+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: add_entry_dag.add_entry_task scheduled__2023-06-15T11:24:00+00:00 [queued]>
[2023-06-15T11:25:01.456+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: add_entry_dag.add_entry_task scheduled__2023-06-15T11:24:00+00:00 [queued]>
[2023-06-15T11:25:01.456+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2023-06-15T11:25:01.457+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2023-06-15T11:25:01.457+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2023-06-15T11:25:01.464+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): add_entry_task> on 2023-06-15 11:24:00+00:00
[2023-06-15T11:25:01.468+0000] {standard_task_runner.py:55} INFO - Started process 4336 to run task
[2023-06-15T11:25:01.471+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'add_entry_dag', 'add_entry_task', 'scheduled__2023-06-15T11:24:00+00:00', '--job-id', '137', '--raw', '--subdir', 'DAGS_FOLDER/postgres_dag.py', '--cfg-path', '/tmp/tmpmdb7a4gs']
[2023-06-15T11:25:01.473+0000] {standard_task_runner.py:83} INFO - Job 137: Subtask add_entry_task
[2023-06-15T11:25:01.524+0000] {task_command.py:389} INFO - Running <TaskInstance: add_entry_dag.add_entry_task scheduled__2023-06-15T11:24:00+00:00 [running]> on host ff7147a0d5a7
[2023-06-15T11:25:01.572+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=add_entry_dag
AIRFLOW_CTX_TASK_ID=add_entry_task
AIRFLOW_CTX_EXECUTION_DATE=2023-06-15T11:24:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-15T11:24:00+00:00
[2023-06-15T11:25:01.581+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-06-15T11:25:01.586+0000] {sql.py:364} INFO - Running statement: INSERT INTO assignment_table (execution_time) VALUES ('2023-06-15 11:25:01.573707'), parameters: None
[2023-06-15T11:25:01.587+0000] {sql.py:373} INFO - Rows affected: 1
[2023-06-15T11:25:01.588+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-06-15T11:25:01.596+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=add_entry_dag, task_id=add_entry_task, execution_date=20230615T112400, start_date=20230615T112501, end_date=20230615T112501
[2023-06-15T11:25:01.645+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2023-06-15T11:25:01.659+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
